Traducir.Api/Controllers/StringsController.cs

```diff
@@ -217,5 +217,28 @@ void composePredicate(Func<SOString, bool> newPredicate)
             return BadRequest();
         }
 
+        [HttpDelete]
+        [Authorize(Policy = "CanSuggest")]
+        [Route("app/api/suggestions{suggestionId:INT}")]
+        public async Task<IActionResult> DeleteSuggestion([FromQuery] int suggestionId)
+        {
+            //Get the suggestion
+            var suggestion = await _soStringService.GetStringSuggestionByIdAsync(suggestionId);
+            if (suggestion == null)
+            {
+                return BadRequest();
+            }
+            if (suggestion.CreatedById != User.GetClaim<int>(ClaimType.Id))
+            {
+                return BadRequest();
+            }
+            var success = await _soStringService.DeleteSuggestionAsync(suggestionId,suggestion.CreatedById);
```

Comment:
if you remove all the lines before here, you can just pass `User.GetClaim<int>(ClaimType.Id)`